desk/app/channels.hoon

```diff
@@ -17,14 +17,14 @@
 /+  channel-json
 ::
 ::  compile all marks
-::XX this is not actually compiling??
+::XX  /~ rune is broken
 :: /~  marks  *  /mar/channel
```

Comment:
Might be useful to point to the accompanying issue here, [`urbit/urbit#5242`](https://github.com/urbit/urbit/issues/5242), so that we know when we can sanely/fully enable this again. (Don't think I had linked that to you before, so see also that issue for further deets.)